app/src/main/java/app/kawamura/kawachi/original/CustomAdapter.kt

```diff
@@ -0,0 +1,32 @@
+package app.kawamura.kawachi.original
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class CustomAdapter( private val animalList: List<Picture>): RecyclerView.Adapter<CustomAdapter.ViewHolder>() {
```

Comment:
```suggestion
class CustomAdapter( private val pictureList: List<Picture>): RecyclerView.Adapter<CustomAdapter.ViewHolder>() {
```
animalList のままになっている！！